pkg/pulumiyaml/run.go

```diff
@@ -318,7 +329,7 @@ func RunTemplate(ctx *pulumi.Context, t *ast.TemplateDecl, loader PackageLoader)
 	}
 
 	// runtime evaluation here
-	diags.Extend(r.Evaluate(ctx)...)
+	diags.Extend(r.Evaluate(ctx, config)...)
```

Comment:
Why does `Evaluate` need a copy of `config` when the config has already been added to the runner with `r.setIntermediates` (L315)?

Reply:
Leftover code, thanks for catching